README.md

```diff
@@ -54,6 +56,22 @@ $ sudo make install
 $ volk_profile
 ```
 
+## Supported platforms
+
+VOLK aims to be portable to as many platforms as possible. We can only run tests on some platforms.
+
+### Hardware architectures
+Currently VOLK aims to run with optimized kernels on x86 with SSE/AVX and ARM with NEON.
+
+### OS / Distro
+We run tests on a variety of Ubuntu versions and aim to support as many current distros as possible.
+The same goal applies to differenct OSes. Although this does only happen rarely, it might occur that VOLK does not work on obsolete distros, e.g. Ubuntu 12.04.
+
+### Compilers
+We want to make sure VOLK works with C/C++ standard compliant compilers. Most notably GCC and Clang.
+Other compilers that fail to be standard compliant, e.g. MSVC or AppleClang, do also receive attention but some functionality might be missing with these compilers due to lack of standard support.
```

Comment:
VOLK builds using modern AppleClang, which correctly reports SIMD support. AppleClang is also C/C++ standards complaint.

MSVC is standards compliant, too, just as much as GCC or *Clang.

Most compilers provide non-standard features, e.g., `gnu++` for GCC, which is only partly supported on Clang or MSVC -- no guarantees!

I'd say to rewrite these 2 sentences.

Reply:
It would be great to tell our CI to use modern AppleClang then and build with proper SIMD support. Anyways, I'm happy to hear the situation is better than I thought according to CI.

I have [compiler support](https://en.cppreference.com/w/cpp/compiler_support) in mind whenever I refer to "standard compliant". Though, this is missing C language features. And I couldn't find such a table for C.
Anyways, I'm still very much disappointed of the lack of support for `aligned_alloc` in MSVC. Of course this frustrating experience motivated this sentence. 
I can rewrite this section to stress that we want to make sure, VOLK works best with open source compilers. What's the status of AppleClang? Are Apples' changes also open source just like Clang? For MSVC, I'd like to point out that it lacks `aligned_alloc` support but works otherwise.

Reply:
CMake can't find `xgetbv` on AppleClang and thus only generic kernels are compiled in our CI. A quick search revealed some discussion around that exact issue from 2013 by @michaelld . How did the situation evolve since then?